tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/elytron/CertificateAuthorityAccountTest.java

```diff
@@ -0,0 +1,235 @@
+package org.jboss.hal.testsuite.test.runtime.elytron;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.SynchronousQueue;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.dmr.ModelNode;
+import org.jboss.hal.dmr.ModelDescriptionConstants;
+import org.jboss.hal.resources.Ids;
+import org.jboss.hal.testsuite.Console;
+import org.jboss.hal.testsuite.Random;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.creaper.command.BackupAndRestoreAttributes;
+import org.jboss.hal.testsuite.fragment.AddResourceDialogFragment;
+import org.jboss.hal.testsuite.page.runtime.elytron.ElytronRuntimeSSLPage;
+import org.jboss.hal.testsuite.test.configuration.elytron.ElytronFixtures;
+import org.jboss.hal.testsuite.util.TestsuiteEnvironmentUtils;
+import org.jboss.hal.testsuite.util.audit.log.AuditLog;
+import org.jboss.hal.testsuite.util.audit.log.AuditLogWatcher;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+@RunWith(Arquillian.class)
+public class CertificateAuthorityAccountTest {
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final Operations operations = new Operations(client);
+    private static final Address AUDIT_ADDRESS = Address.coreService("management").and("access", "audit");
+    private static final Address AUDIT_LOG_ADDRESS = AUDIT_ADDRESS.and("logger", "audit-log");
+    private static final Address JSON_FORMATTER_ADDRESS = AUDIT_ADDRESS.and("json-formatter", "json-formatter");
+    private static final Address FILE_HANDLER_ADDRESS = AUDIT_ADDRESS.and("file-handler", "file");
+    private static final BackupAndRestoreAttributes backup =
```

Comment:
I would suggest renaming the `backup` field to something like `auditLogAttributesBackup` to avoid confusion with (may be expected) `org.wildfly.extras.creaper.commands.foundation.online.SnapshotBackup` when reading following `backup` field usage.